test/qa/test_meshnet.py

```diff
@@ -317,4 +319,147 @@ def test_account_switch():
     # Recover starting state (this is the simplest way)
     teardown_module(None)
     setup_module(None)
-    
\ No newline at end of file
+
+
+def test_invite_send_repeated():
+
+    with lib.Defer(lambda: sh.nordvpn.meshnet.invite.revoke("test@test.com")):
+        meshnet.send_meshnet_invite("test@test.com")
+
+        with pytest.raises(sh.ErrorReturnCode_1) as ex:
+            meshnet.send_meshnet_invite("test@test.com")
+
+        assert "Meshnet invitation for 'test@test.com' already exists." in str(ex.value)
+
+
+def test_invite_send_own_email():
+    
+    with pytest.raises(sh.ErrorReturnCode_1) as ex:
+        meshnet.send_meshnet_invite(os.environ.get("DEFAULT_LOGIN_USERNAME"))
+
+    assert "Email should belong to a different user." in str(ex.value)
+
+
+def test_invite_send_not_an_email():
+
+    with pytest.raises(sh.ErrorReturnCode_1) as ex:
+        meshnet.send_meshnet_invite("test")
+
+    assert "Invalid email 'test'." in str(ex.value)
+
+
+@pytest.mark.skip(reason="A different error message is expected - LVPN-262")
+def test_invite_send_long_email():
+    
+    # A long email address containing more than 256 characters is created
+    email = "test" * 65 + "@test.com"
+
+    with pytest.raises(sh.ErrorReturnCode_1) as ex:
+        meshnet.send_meshnet_invite(email)
+
+    assert not "It's not you, it's us. We're having trouble with our servers. If the issue persists, please contact our customer support." in str(ex.value)
+
+
+@pytest.mark.skip(reason="A different error message is expected - LVPN-262")
+def test_invite_send_email_special_character():
+    
+    with pytest.raises(sh.ErrorReturnCode_1) as ex:
+        meshnet.send_meshnet_invite("\u2222@test.com")
+
+    assert not "It's not you, it's us. We're having trouble with our servers. If the issue persists, please contact our customer support." in str(ex.value)
```

Comment:
Same as with too long email - app should respond with clear message about invalid email.